answers/Node.md

```diff
@@ -0,0 +1,4 @@
+* Describe the global variable `process` as part of the node runtime
+The process object is a global that provides information about, and control
+over, the current Node.js process. As a global, it is always available to
+Node.js applications without using require().
```

Comment:
Let's use backticks (`) around anything that is actual code so that it looks like `require()` and `process`